tools/qualification-tool/pom.xml

```diff
@@ -154,7 +154,9 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.21.0</version>
+        <configuration>
+          <skipTests>false</skipTests>
```

Comment:
@yaooqinn, could you explain why we need this?

Reply:
FYI, https://github.com/apache/incubator-gluten/actions/runs/13717946244/job/38366738976

One of the reasons why we need to disable the surefire-plugin is that one of our CI job failed w/ unknown parameters